backend/coins/btc/account.go

```diff
@@ -137,13 +140,13 @@ func NewAccount(
 	log.Debug("Creating new account")
 
 	account := &Account{
-		BaseAccount:    accounts.NewBaseAccount(config, coin, log),
-		coin:           coin,
-		dbSubfolder:    "", // set in Initialize()
-		forceGapLimits: forceGapLimits,
-
-		log:        log,
-		httpClient: httpClient,
+		BaseAccount:                     accounts.NewBaseAccount(config, coin, log),
+		coin:                            coin,
+		dbSubfolder:                     "", // set in Initialize()
+		forceGapLimits:                  forceGapLimits,
+		getAddressFromDifferentAccounts: getAddress,
```

Comment:
why different names for the struct field and param name? By looking at the actual callback, it returns addresses not only from different accounts, but also from the same account, so the name is a bit confusing.

Reply:
I totally agree, I found it hard to name things in this PR :) I went for "GetAddressFromSameKeystore" (based on another change you suggested) but open to different naming